packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -181,6 +162,50 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
     };
   }, [handleSubmit, submissionNotifier]);
 
+  const CustomDatePicker = ({
+    id,
+    maxDate,
+    minDate,
+    onChange,
+    value,
+    inputId,
+    inputLabelText,
+  }: CustomDatePickerProps) => {
+    return (
+      <>
+        {isTablet ? (
```

Comment:
Since the inner one is pretty much the same I'd say ideally you introduce a component like

```jsx
const ResponsiveWrapper = ({ isTablet, children }) => {
  if (isTablet) {
    return (
      <Layer>{children}</Layer>
    );
  }

  return <>{children}</>;
};
```

and use it in here like

```jsx
        <ResponsiveWrapper isTablet={isTablet}>
            <DatePicker
              id={id}
              datePickerType="single"
              dateFormat="d/m/Y"
              minDate={minDate}
              maxDate={maxDate}
              placeholder="dd/mm/yyyy"
              onChange={onChange}
              value={value}
            >
              <DatePickerInput id={inputId} labelText={inputLabelText} />
            </DatePicker>
          </ResponsiveWrapper >
```

Reply:
Lovely, thanks Florian!